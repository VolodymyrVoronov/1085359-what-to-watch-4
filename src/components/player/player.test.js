import React from "react";
import renderer from "react-test-renderer";
import Player from "./player.jsx";

const TEST_ITEMS = {
  ID: 0,
  IS_ACTIVE: false,
  POSTER: ``,
  SRC: ``,
};

const HANDLE_EVENT = () => {};

it(`render should be match markup`, () => {
  const result = renderer
    .create(<Player
      id={TEST_ITEMS.ID}
      isActive={TEST_ITEMS.IS_ACTIVE}
      onPlay={HANDLE_EVENT}
      onEnd={HANDLE_EVENT}
      poster={TEST_ITEMS.POSTER}
      src={TEST_ITEMS.SRC}
    />, {
      createNodeMock: () => {
        return {};
      }
    })
    .toJSON();

  expect(result).toMatchSnapshot();
});
