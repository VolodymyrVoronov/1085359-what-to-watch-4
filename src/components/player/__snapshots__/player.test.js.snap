// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should be match markup 1`] = `
<video
  className="player__video"
/>
`;
